app/src/main/java/hm/orz/chaos114/android/androidsaddlersample/MainActivity.java

```diff
@@ -13,6 +13,10 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        if (savedInstanceState == null) {
+            new IllegalArgumentException("input must not be null");
```

Comment:
ERROR: [CORRECTNESS][RV_EXCEPTION_NOT_THROWN] new IllegalArgumentException(String) not thrown in hm.orz.chaos114.android.androidsaddlersample.MainActivity.onCreate(Bundle)
http://findbugs.sourceforge.net/bugDescriptions.html#RV_EXCEPTION_NOT_THROWN